tests/conftest.py

```diff
@@ -9,11 +9,13 @@
 
 import docker
 import flask
-import ostorlab
 import pytest
 import redis
 from docker.models import networks as networks_model
-from flask import testing
+from flask import testing as flask_testing
```

Comment:
why ?

Reply:
`conftest.py` is located in the root of the testing folder and is not specific to any single module. Naming it simply "testing" is confusing. The same applies to naming conventions for "client" and other elements.

By using specific and descriptive names, we can easily understand the utilities of something either by its path or its name. Since the name is global, it should be more specific for better readability.